README.md

```diff
@@ -1,2 +1,18 @@
 # stacks-java
 Java Web API Application Scaffolding for Amido Stacks
+
+## Run Locally
+
+> ./gradlew bootRun
+
+## Build Docker Image
+
+> docker build -t <image-tag> .
+
+## Swagger/OAS
+
+Automatically generated. Go to: http://localhost:9000/api-docs
+
+## Health check
+
+Available at: http://localhost:9001/actuator/health
```

Comment:
Likewise can we match .NET for health endpoint?

Reply:
We can do this. `/health` is what is being used. Easy config change

Reply:
done